polarizer/win32/resources.rc

```diff
@@ -1,6 +1,7 @@
-
 destroy-fx-link.png  PNG DISCARDABLE  "..\\gui\\graphics\\destroy-fx-link.png"
 implode-button.png  PNG DISCARDABLE  "..\\gui\\graphics\\implode-button.png"
 polarizer-background.png  PNG DISCARDABLE  "..\\gui\\graphics\\polarizer-background.png"
 slider-background.png  PNG DISCARDABLE  "..\\gui\\graphics\\slider-background.png"
 slider-handle.png  PNG DISCARDABLE  "..\\gui\\graphics\\slider-handle.png"
+
+px10.ttf  DFX_TTF DISCARDABLE  "..\\..\\fonts\\bboron.ttf"

```

Comment:
Wouldn't we want to embed this as bboron.ttf? (first token on the line)

Reply:
oops fixed!